src/hid-sensor.cpp

```diff
@@ -339,6 +348,29 @@ uint32_t hid_sensor::fps_to_sampling_frequency( rs2_stream stream, uint32_t fps
     else
         return fps;
 }
+void hid_sensor::set_imu_sensitivity( rs2_stream stream, float value ) 
+{
+    _imu_sensitivity_per_rs2_stream[stream] = value;
+}
+
+void hid_sensor::set_gyro_scale_factor(double scale_factor) 
+{
+    _hid_device->set_gyro_scale_factor( scale_factor );
+}
+
+/*For gyro sensitivity - FW expects 0/0.1/0.2/0.3/0.4 we convert the values from the enum 0/1/2/3/4
+the user chooses to the values FW expects using gyro_sensitivity_convert*/
+double hid_sensor::get_imu_sensitivity_values( rs2_stream stream )
+{
+    if( _imu_sensitivity_per_rs2_stream.find( stream ) != _imu_sensitivity_per_rs2_stream.end() )
+    {
+        return gyro_sensitivity_convert.at( _imu_sensitivity_per_rs2_stream[stream] );
+    }
+    else
+        //gyro sensitivity default value is +-1000 therefore sending 0.1 
+        //accel sensitivity default value is +-4g therefore sending 0.001
```

Comment:
Please edit the comment. The scale factor connection to the dynamic range is not clear.

Reply:
Done